blog/ca/3_web-scraping-de-feines-a-linkedIn-utilitzant-puppeteer-i-rxjs.mdx

```diff
@@ -0,0 +1,502 @@
+---
+title: Web Scraping d'ofertes de treball a LinkedIn utilitzant Puppeteer i RxJS
+description: Tutorial sobre com extreure ofertes de treball de LinkedIn utilitzant Puppeteer i RxJS
+image: /assets/blog/web-scraping-de-feines-a-linkedin-utilitzant-puppeteer-i-rxjs/cover-image.png
+published: true
+publishedDate: 2023-10-03
+authors: llorenspujol
+seo:
+  metatitle: >
+    Web Scraping d'ofertes de treball a LinkedIn utilitzant Puppeteer i RxJS
+  metadescription: >
+    Tutorial sobre com extreure ofertes de treball de LinkedIn utilitzant Puppeteer i RxJS
+  image: >
+    /assets/blog/web-scraping-de-feines-a-linkedin-utilitzant-puppeteer-i-rxjs/cover-image.png
+---
+
+El web scraping pot semblar una tasca senzilla, però hi ha molts reptes a superar. En aquest blog, ens endinsarem en com fer scraping a LinkedIn per extreure llistats de treball. Per fer això, utilitzarem [Puppeteer](https://pptr.dev/) i [RxJS](https://rxjs.dev/). L'objectiu és assolir web scraping d'una manera declarativa, modular i escalable.
+
+## Entenent el Web scraping
+
+El web scraping és una tècnica d'extracció de dades utilitzada per recopilar informació de llocs web. Involucra el procés automatitzat d'obtenció de dades específiques de pàgines web, com ara text, imatges, enllaços i més, i després emmagatzemar o processar aquestes dades per a diversos propòsits.
```

Comment:
I have a couple of questions here:

I'm not sure if "Involucra" is the best word here. I'd rather use something like "Es tracta d'un..." or "Consisteix en"
"Obtenció" should be "extracció"
"textos" instead of "text"
"I després" can be "per després/per posteriorment"

Reply:
Cert